### ISSUE TITLE
Not all fields from `EthereumTesterProvider` are being properly remapped.

### ISSUE THREAD

**Issue by pipermerriam** [ethereum/web3.py#325]
* Version: 3.15.0

### What was wrong?

The `web3.providers.eth_tester.middleware.ethereum_tester_middleware` is not converting the key `log_index` to the web3 naming convention `logIndex`.

### How can it be fixed?

Add a new `log_entry_remapper` function in the same style as the `transaction_params_remapper` which maps the field `log_index > logIndex`.

**Comment by carver** [comment]
Probably will be resolved by the time #530 is complete.

**Comment by dylanjw** [comment]
This looks like it is working now:

```
(Pdb) web3.eth.getTransactionReceipt("0x0b5b9f20bf52b68a8b262755fd84d711ecf8d58fdd8e64a69e5d90b0492a7bf2")
AttributeDict({'transactionHash': HexBytes('0x0b5b9f20bf52b68a8b262755fd84d711ecf8d58fdd8e64a69e5d90b0492a7bf2'), 'transactionIndex': 0, 'blockNumber': 3, 'blockHash': HexBytes('0x87e8aeb7494b4d70ceacca741920eb1e67377c838a4744a34cedcb7da2127a92'), 'cumulativeGasUsed': 22438, 'gasUsed': 22438, 'contractAddress': None, 'logs': [AttributeDict({'type': 'mined', 'logIndex': 0, 'transactionIndex': 0, 'transactionHash': HexBytes('0x0b5b9f20bf52b68a8b262755fd84d711ecf8d58fdd8e64a69e5d90b0492a7bf2'), 'blockHash': HexBytes('0x87e8aeb7494b4d70ceacca741920eb1e67377c838a4744a34cedcb7da2127a92'), 'blockNumber': 3, 'address': '0xF2E246BB76DF876Cef8b38ae84130F4F55De395b', 'data': '0x', 'topics': [HexBytes('0x1e86022f78f8d04f8e3dfd13a2bdb280403e6632877c0dbee5e4eeb259908a5c')]})], 'status': 1})
(Pdb) web3.providers
[<web3.providers.eth_tester.main.EthereumTesterProvider object at 0x7f8cfc2ce9b0>]
(Pdb) 
```
